### ISSUE TITLE
Using a successfully Linux build .aar file in Windows

### ISSUE THREAD

**Issue by Wayne-nulitics** [homuler/MediaPipeUnityPlugin#115]
Successfully built the plugin for Android using a Linux machine.
Successfully built the plugin for Windows CPU using a Windows 10 machine.

Want to copy the Android files from Packages\com.github.homuler.mediapipe\Runtime\Plugins from the Linux machine to the Windows one to Build Android apps on Windows in Unity using pre-built plugin.

This keeps on failing due to not finding the correct opencv files.

We have another SDK that only works in a Windows Editor that we want to compile with Mediapipe to Android via Unity.

TLDR, is there a way to use .aar files/libraries build (and working) on a Linux system on a Windows system?

**Comment by homuler** [comment]
Please use [build-installation-issue.md](https://github.com/homuler/MediaPipeUnityPlugin/blob/master/.github/ISSUE_TEMPLATE/build-installation-issue.md) as the issue template, and describe precisely what kind of error occurs on which device.

In general, you can use AARs built on Linux (they are only loaded on Android devices, and not on your UnityEditor).

**Comment by Wayne-nulitics** [comment]
OK, got this working, you can build the .aar file on a Linux machine, then switch to a Windows machine, change Platform to Android and build, you just can't use it in Editor mode.
